wremnants/CardTool.py

```diff
@@ -301,23 +305,30 @@ def addSystematic(self, name, systAxes=[], systAxesFlow=[], outNames=None, skipE
         # for now better not to use the options, although it might be useful to keep it implemented
         if mirrorDownVarEqualToUp or mirrorDownVarEqualToNomi:
             raise ValueError("mirrorDownVarEqualToUp and mirrorDownVarEqualToNomi currently lead to pathological results in the fit, please keep them False")
-        
+
+        if not mirror and (mirrorDownVarEqualToUp or mirrorDownVarEqualToNomi):
+            raise ValueError("mirrorDownVarEqualToUp and mirrorDownVarEqualToNomi requires mirror=True")
+
+        if mirrorDownVarEqualToUp and mirrorDownVarEqualToNomi:
+            raise ValueError("mirrorDownVarEqualToUp and mirrorDownVarEqualToNomi cannot be both True")
+
+        if symmetrize not in ["average", "conservative"]:
```

Comment:
symmetrize is actually supposed to be allowed to be None (even if it's not the default)